src/app/options/qgsoptions.cpp

```diff
@@ -2594,6 +2598,19 @@ void QgsOptions::moveLocalizedDataPathDown()
   }
 }
 
+void QgsOptions::alwaysUseDecimalPointChanged( bool checked )
+{
+  // In the Measure Tool copy feature,
+  // comma is only allowed as field separator if the locale do not use it as decimal separator
+  // or if the user has overridden the decimal separator to always be a point
+  mSeparatorComma->setEnabled( checked || QLocale().decimalPoint() != QStringLiteral( "," ) );
```

Comment:
```suggestion
  mSeparatorComma->setEnabled( checked || QLocale().decimalPoint() != QChar( ',' ) );
```

I kind of remember this was more efficient but I'm not 100% sure.